doc/source/whatsnew/v3.0.0.rst

```diff
@@ -367,13 +367,17 @@ The inferred resolution now matches that of the input strings:
     In [4]: pd.to_datetime(["2024-03-22 11:43:01.002003"]).dtype
     In [5]: pd.to_datetime(["2024-03-22 11:43:01.002003004"]).dtype
 
+This is also a change for the :class:`Timestamp` constructor with a string input, which in version 2.x.y could give second or millisecond unit, which users generally disliked (:issue:`52653`)
```

Comment:
Just want to express, that over at xarray we actually liked that behaviour. That change just broke our nightly tests.